src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiResponseGenerator.cs

```diff
@@ -33,6 +33,8 @@ internal static class OpenApiResponseGenerator
                 },
 
                 { Constants.StatusCode204, new OpenApiResponse { Description = "Success"} },
+                { Constants.StatusCode201, new OpenApiResponse { Description = "Created"} },
+                { Constants.StatusCodeClass2XX, new OpenApiResponse { Description = "Success"} },
```

Comment:
I'm confused, why are we adding the range and the specific codes? shouldn't it be one or the others?

Reply:
It's just a dictionary that we can use to extract the response object for the status code range 2XX for scenarios where `addNoContent` == `true` but `schema` == `null`. 
We'll end up with:
![image](https://user-images.githubusercontent.com/40403681/226277517-3bf16d7c-58ee-4c91-8944-632309f37d9e.png)